ChatQnA/README.md

```diff
@@ -18,17 +18,50 @@ This ChatQnA use case performs RAG using LangChain, Redis VectorDB and Text Gene
 
 The ChatQnA service can be effortlessly deployed on either Intel Gaudi2 or Intel XEON Scalable Processors.
 
+Before deploying ChatQnA services, you need to set up environment variables.
```

Comment:
To set up environment variables for deploying ChatQnA services, follow these steps:

1. Set the required environment variables:
   ```bash
   export host_ip="External_Public_IP"
   export HUGGINGFACEHUB_API_TOKEN="Your_Huggingface_API_Token"
   ```

2. If you are in a proxy environment, also set the proxy-related environment variables:
   ```bash
   export no_proxy="Your_No_Proxy"
   export http_proxy="Your_HTTP_Proxy"
   export https_proxy="Your_HTTPS_Proxy"
   ```